luaui/Widgets/camera_flip.lua

```diff
@@ -29,14 +29,19 @@ local function cameraFlipHandler()
 	-- CardinalLock messes up rotation
 	local previousLock = Spring.GetConfigInt("CamSpringLockCardinalDirections")
 
-	if previousLock == 1 then
-		Spring.SetConfigInt("CamSpringLockCardinalDirections", 0)
+	if previousLock == 0 then
+		Spring.SetConfigInt("CamSpringLockCardinalDirections", 1)
+	end
+
+	if camState.ry > 0 then
+		camState.ry = camState.ry - math.pi - 1 / 3
```

Comment:
what's the 1/3 constant?

Reply:
A value larger than the cardinal lock width of .2